### ISSUE TITLE
Handling time zones for Date, Time and DateTime fields

### ISSUE THREAD

**Issue by joshua-sharp** [unite-cms/unite-cms#723]
The Time, Date and DateTime fields store their values directly in the database without accounting for any potential difference in each user's preferred time zone. If multiple UniteCMS users in different time zones are updating content that use these fields, the values should be stored in a format that ensures date & time entries result in the correct values regardless of the time zone.

Another feature that would be useful when utilising these fields, is allowing a user to set their preferred time zone. It would be useful to have it set by an individual user, or having the time zone set for an organization.

Also when retrieving a stored date and time value and comparing it with another, a standardised format such as storing all values in UTC would prevent any later comparisons from being incorrect.

Please let me know if you have something else in mind for this problem.

**Comment by franzwilding** [comment]
You are right!

The new date picker is based on https://github.com/charliekassel/vuejs-datepicker#readme. Is your feature request possible with this lib?

**Comment by joshua-sharp** [comment]
Yes, the vue-datepicker package has a prop *use-utc* which will allow the user's current time to be stored as UTC in the database. However, this library appears to use the user's browser to determine the current time. Though this would meet my requirements, I expect there may be other users who would want to be able to use a different timezone.

Allowing a user to set a preferred timezone timezone different to their current browser's local timezone would require the application to support a user timezone setting that would then need to be respected by the Time, Date and DateTime Vue components.
